File: go_back/internal/model/feature_card.go
package model

type FeatureCard struct {
	ID            uint64 `gorm:"primaryKey" json:"id"`
	Title         string `gorm:"column:title;size:255;not null" json:"title"`
	TitleEn       string `gorm:"column:title_en;size:255" json:"titleEn"`
	Description   string `gorm:"column:description;size:500" json:"description"`
	DescriptionEn string `gorm:"column:description_en;size:500" json:"descriptionEn"`
	Icon          string `gorm:"column:icon;size:100" json:"icon"`
	Gradient      string `gorm:"column:gradient;size:100" json:"gradient"`
	DisplayOrder  int    `gorm:"column:display_order;default:0" json:"displayOrder"`
	Enabled       bool   `gorm:"column:enabled;default:true" json:"enabled"`
}

func (FeatureCard) TableName() string {
	return "feature_cards"
}

File: backend/src/main/java/com/personal/website/entity/FeatureCard.java
package com.personal.website.entity;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name = "feature_cards")
public class FeatureCard {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false)
    private String title;

    private String titleEn;

    @Column(length = 500)
    private String description;

    @Column(length = 500)
    private String descriptionEn;

    private String icon = "Code";

    private String gradient = "from-blue-500 to-cyan-500";

    private Integer displayOrder = 0;

    private Boolean enabled = true;
}
